### ISSUE TITLE
Mic Disabled (Conversation Error)

### ISSUE THREAD

**Issue by JTD420** [eouia/MMM-AssistantMk2#121]
**Describe the problem**
If I use the call word to speak to the assistant but don't speak clearly or for whatever reason the assistant is unable to pickup what was said I receive the `Conversation_Error` message which then results in me being unable to use the voice commands until I restart MM with pm2.

**To Reproduce**
Use the voice command to get the assistant to start listening to you.
Then either say nothing or try saying something and stopping halfway through, maybe interrupting yourself with music etc. 
Anything to get the assistant to not pickup what you've said I think.

**Environment:**
 - Device : Raspberry Pi 3b+ 
 - OS: Raspbian Stretch Desktop
 - Node Version : 8.11.1

I am using PM2 however because everything seemingly works unless somethings not detected im not sure you need any more info from me such as logs?

What is the expected behavior of `Conversation_Error` I'd have thought presumably it should just beep like it does when it hears the call word to signify it's stopped listening and then you can try again...?

**Comment by eouia** [comment]
When no-input(including mic error, or not clear input) which cannot be handled by assistant cloud server, that error will return.
Usually, it is better to ignore it. conv error message will be disappeared after some sec then be ready to activate  again unless it is mic h/w issue.

**Comment by JTD420** [comment]
> When no-input(including mic error, or not clear input) which cannot be handled by assistant cloud server, that error will return.
> Usually, it is better to ignore it. conv error message will be disappeared after some sec then be ready to activate again unless it is mic h/w issue.

I have tried waiting but after the message disappears it doesn't allow to be activated again without restarting the MM..

It's not the mic as it works fine constantly unless it doesn't pickup my voice such as when I'm a bit too far away. 
I'm using my DSLR microphone for now so I know the hardware is good, it's this mic: https://www.amazon.co.uk/Rode-VideoMicro-Compact-Camera-Microphone/dp/B015R0IQGW/

What do you think could be the issue?

**Comment by eouia** [comment]
If so, your configuration might be wrong, so it cannot get proper input from mic.
-check your mic device name with “arecord -l” and “arecord -D [YOUR_DEVICE_NAME]” and see the “record.device” in your AMK2 configuration section.

**Comment by JTD420** [comment]
> If so, your configuration might be wrong, so it cannot get proper input from mic.
> -check your mic device name with “arecord -l” and “arecord -D [YOUR_DEVICE_NAME]” and see the “record.device” in your AMK2 configuration section.

Thank's for your help. Not really sure what the issue was tbh! I ended up reinstalling but using Rasbian instead of the full version and that seemed to fix all of my errors I was getting 🤷‍♂️ 

I think I may have had the issue happen again since but I closed out of NPM before I got a chance to check the error so I'll close this and re-open if it becomes an issue again!
